docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -282,11 +282,57 @@ be sampled such that:
 	# (s is the shape parameter)
 	response_time_distribution = lognorm(s=sigma, scale=median)
 
-**Coverage Data**
+**Population Coverage Data**
+
+The coverage algorithm for vitamin A fortification should follow the same approach described in this concept model document for folic acid fortification (see `Population Coverage Data`_).
+
+.. list-table:: Means and 95% CI's for existing population coverage of vitamin A fortification (% of total population)
+  :widths: 5 5 5 5
+  :header-rows: 1
+
+  * - Location
+    - :math:`a` = Eats fortified vehicle
+    - :math:`b` = Eats fortifiable vehicle
+    - :math:`c` = Eats vehicle
+  * - Ethiopia
+    - 1.0 (see below)
+    - 44 (34, 54)
+    - 55 (45, 65)
+  * - India
+    - 24.3 (21.1, 27.9)
+    - 89.4 (87.0, 91.8)
+    - 100 (100, 100)
+  * - Nigeria (Kano)
+    - 7.6 (5.9, 9.4)
+    - 35.9 (32.7, 39.1)
+    - 98.4 (97.6, 99.3)
+  * - Nigeria (Lagos)
+    - 7.2 (5.5, 8.9)
+    - 22.7 (19.9, 25.5)
+    - 98.6 (97.8, 99.3)
+
+For all values other than :math: `a` for Ethiopia, use a Beta distribution 
+with mean equal to the central estimate, and variance equal to the variance of 
+a normal distribution with the same mean and 95% confidence interval. 
```

Comment:
I haven't actually verified that this approach works yet -- James seemed to think there was a problem with it.

Reply:
I'm about to try to figure this out.

Reply:
I don't think there's a problem.  I was just surprised how different the plots looked for your stand in distribution vs the one where you have data.  But I also did a weird thing to squeeze the data into a range.